### ISSUE TITLE
[Request] Add a output file option for the export command

### ISSUE THREAD

**Issue by NyanKiyoshi** [sdispater/poetry#1015]
- [x] I have searched the [issues](https://github.com/sdispater/poetry/issues) of this repo and believe that this is not a duplicate.
- [x] I have searched the [documentation](https://poetry.eustace.io/docs/) and believe that my question is not covered.

## Feature Request
Hello, I think it would be very handy to have a `-o` parameter in poetry's `export` command so we could be able to tell poetry to which file we want to export the requirements, independently from the format (which as of now, the only format supported is `requirements.txt`).

So we could prevent doing such things:
```
poetry export --dev -f requirements.txt && mv requirements.txt requirements_dev.txt \
    && poetry export -f requirements.txt
```

Instead, we would have:
```
poetry export --dev -f requirements.txt -o requirements_dev.txt \
    && poetry export -f requirements.txt
```

Also, it would be completely cross platform.

**Comment by bersace** [comment]
Funny. You posted this ticket just 45 minutes after #1014 on the same subject.

**Comment by NyanKiyoshi** [comment]
Closed by #1035, thanks @richin13!
